backend/app/views/backend/activities/new.html.erb

```diff
@@ -1,14 +0,0 @@
-<div class="l-navigation">
```

Comment:
Thinking about removing the new templates, I think it should be OK as you say.
The only reason they are kept normally if there's a different title or something, but I see that this is handled inside the form, for instance here: https://github.com/Vizzuality/grid-arendal/blob/feature/CMS-news-section/backend/app/views/backend/news_articles/_form.html.slim#L1 You have:

`== render 'backend/shared/form/header', form: form, title_placeholder: 'New News Article'`

But I think I'd at least replace "New News Article" with "News article title", so that even if you are editing a title the placeholder makes sense.